pkg/minikube/image/cache.go

```diff
@@ -60,7 +60,11 @@ func SaveToDir(images []string, cacheDir string) error {
 			dst := filepath.Join(cacheDir, image)
 			dst = localpath.SanitizeCacheDir(dst)
 			if err := saveToTarFile(image, dst); err != nil {
-				klog.Errorf("save image to file %q -> %q failed: %v", image, dst, err)
+				klog.Errorf("save image to file %q -> %q failed: \n" +
```

Comment:
this doesnt help the user in the UI, Suggest instead of logging klog.Errr we should add out.Warn with simnple english (an no long error)

out.WarningT("The image you are trying to add (NAME_OF_IMAGE) doesn't exist ! ")

Reply:
and also I recommend creating an ErrorType simmilar to https://github.com/medyagh/minikube/blob/ff0e25ada888cf1bd7b262857ce96531c1231e69/pkg/drivers/kic/oci/errors.go#L37

so you can return a specific Error type in case of cache not existing
and then you can chceck if errr == ErrCacheImageDoesntExist {
out.WarningT("The image you are trying to add (NAME_OF_IMAGE) doesn't exist ! ")
} else {
return it like we used to do before and make ugly crash, since we dont know what is going on !
}

Reply:
Thank you for a detailed comment. That really helps a new newcomer(meaning me). I will follow your advice

Reply:
@medyagh Do I understand correctly that I need to create a separate folder named cache in pkg/minikube/ in which cache_errors.go will be created with ErrCacheImageDoesntExist. Or there is a better way to do it?

Reply:
no probolem, if we only have one error, we dont need to create a new file.